src/sensor_state_data/data.py

```diff
@@ -289,8 +289,7 @@ def update_sensor(
     def fire_event(
         self,
         key: str,
-        event_type: str,
-        event_subtype: str | None = None,
+        event_data: str | dict[str, str | int | float],
```

Comment:
Is there any reason to not always make it a `dict`?

Reply:
We can do that, one has to use `{"toggle": None}` if he only wants to send "toggle".

Reply:
I think thats fine. It should keep the consumer code a big simpler to not have to handle both cases

Reply:
Another option might be to leave `event_type` as a `str` and then have `event_properties` be a `dict` so you have some level of normalization and the non-normalized data in the `dict`

Reply:
If that is better, it is also fine with me. Do you prefer that?

Reply:
Done